packages/target-decisioning-engine/src/contextProvider.spec.js

```diff
@@ -333,6 +333,49 @@ describe("contextProvider", () => {
     expect(createMboxContext(undefined)).toEqual({});
   });
 
+  it("supports mbox content with dot notation", () => {
+    expect(
+      createMboxContext({
+        index: 10,
+        name: "dot_mbox",
+        parameters: {
+          "favorite.pizza": "PINEAPPLE",
+          "favorite.month": "august",
+          "ignore..notation": true,
+          "support.nested.notation": true,
+          "support.nested.dots": true,
+          ".best.coast": "east",
+          "trailing.dots.": "bad"
+        }
+      })
+    ).toEqual({
+      "favorite": {
+        pizza: "PINEAPPLE",
+        pizza_lc: "pineapple",
+        month: "august",
+        month_lc: "august"
+      },
+      "ignore..notation": true,
+      "ignore..notation_lc": true,
+      ".best.coast": "east",
+      ".best.coast_lc": "east",
+      "trailing.dots.": "bad",
+      "trailing": {
```

Comment:
i would expect the `trailing.dots.` property to translate over directly rather than also generate a `trailing` object with values.